package.json

```diff
@@ -51,26 +52,24 @@
     "@babel/preset-env": "^7.12.11",
     "broccoli-file-creator": "^2.1.1",
     "broccoli-merge-trees": "^2.0.0",
-    "ceibo": "~2.0.0",
+    "ceibo": "github:ro0gr/ceibo#multi-authoring-format",
+    "ember-auto-import": "^2.2.3",
```

Comment:
@ro0gr just FYI (hope this would help with failures), you may need to bump this to 2.4.0 to get `@embroider/*` 1.0.0 across the board